features/rum/src/appleMain/kotlin/com/datadog/kmp/rum/internal/DDRumMonitorProxy.kt

```diff
@@ -142,6 +143,17 @@ internal interface DDRumMonitorProxy {
         attributes: Map<Any?, *>
     )
 
+    fun startFeatureOperation(name: String, operationKey: String?, attributes: Map<Any?, *>)
+
+    fun succeedFeatureOperation(name: String, operationKey: String?, attributes: Map<Any?, *>)
+
+    fun failFeatureOperation(
+        name: String,
+        operationKey: String?,
+        failureReason: DDRUMFeatureOperationFailureReason,
+        attributes: Map<Any?, *>
```

Comment:
Shouldn't it be `Map<String, Any?>)` ? 
Same for `startFeatureOperation` / `succeedFeatureOperation` ?

Reply:
no, because this is the type generated by KMP for iOS SDK bindings, it is similar for all method in RUM monitor.

<img width="434" height="58" alt="image" src="https://github.com/user-attachments/assets/c945819e-45c6-45b6-92b9-6d533f06c4b3" />